features/upgrade/sdn/externalIP-upgrade.feature

```diff
@@ -0,0 +1,73 @@
+Feature: SDN externalIP compoment upgrade testing
+ 
+  # @author weliang@redhat.com
+  @admin
+  @upgrade-prepare
+  Scenario: Check the externalIP works well after upgrade - prepare
+    Given I switch to cluster admin pseudo user
+    And I run the :new_project client command with:
+      | project_name | externalip-upgrade |
+    Then the step should succeed  
+    Given I store the schedulable nodes in the :nodes clipboard
+    And the Internal IP of node "<%= cb.nodes[0].name %>" is stored in the :hostip clipboard
+
+    # Create additional network through CNO
+    Given as admin I successfully merge patch resource "networks.config.openshift.io/cluster" with:
+      | {"spec":{"externalIP":{"policy":{"allowedCIDRs":["<%= cb.hostip %>/24"]}}}} |
+
+    # Create a svc with externalIP
+    When I use the "externalip-upgrade" project
+    Given I obtain test data file "networking/externalip_service1.json"
+    And I wait up to 500 seconds for the steps to pass:
+    """
+    When I run oc create over "externalip_service1.json" replacing paths:
+      | ["spec"]["externalIPs"][0] | <%= cb.hostip %>    |
+      | n                          | externalip-upgrade  |
+    Then the step should succeed
+    """
+
+    # Create a pod
+    Given I obtain test data file "networking/externalip_pod.yaml"
```

Comment:
@weliang1 If this pod to just to validate the service configured with external ip, is another YAML file (externalip_pod.yaml) needed? Could you not use existing YAML files and replace the pod name to what ever is desired for this test?

Reply:
@asood-rh Good suggestion, let me update my script to be the way as you suggest here.